CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Contributing
```

Comment:
IMO a good contributing guide should have:
1. Local setup (clone repo + npm run dev to start rollup)
2. Running lint and tests
3. PR best practices (describe the feature, add unit tests if applicatble etc)

We can borrow this from someones repo, for example elysia has a nice short and to the point guide: https://github.com/elysiajs/elysia/blob/main/CONTRIBUTING.md

Reply:
I've used it as an inspiration. Provided something initial. We can evlove this over time.